### PR TITLE
linux_homework

### PR DESCRIPTION
**Pull request by cbazdyga**
created by CBazdyga

### CODE REVIEW

**Comment by ziobron** [comment]
@cbazdyga, we do not have your data in the Coders School database.
1. Please fill in this form https://forms.gle/rkzLsMUSBdnGQgdM9
2. Edit this PR to run this automation once again (you can just add a space in the description or title).
3. Wait one minute. If the "Standard" or "Premium" label was added then everything is fine :)
4. Ask @ziobron to add you to the @coders-school/online-2021 group, so that you can approve and run workflows that check your homework. If you are impatient, ask anyone else on Discord to run the workflow for you :)

File: homework/linux.txt [hunk 1]
```diff
@@ -0,0 +1,30 @@
+cezary@cezary-VirtualBox:~/Documents/cpp_workspace$ mkdir dummy_dir
+cezary@cezary-VirtualBox:~/Documents/cpp_workspace$ touch dummy_dir/process.txt
```
Comment (BarTes8) [inline homework/linux.txt:2]: processes.txt

File: homework/linux.txt [hunk 2]
```diff
@@ -0,0 +1,30 @@
+cezary@cezary-VirtualBox:~/Documents/cpp_workspace$ mkdir dummy_dir
+cezary@cezary-VirtualBox:~/Documents/cpp_workspace$ touch dummy_dir/process.txt
+cezary@cezary-VirtualBox:~/Documents/cpp_workspace$ ps aux >> dummy_dir/process.txt 
+cezary@cezary-VirtualBox:~/Documents/cpp_workspace$ cp dummy_dir/process.txt dummy_dir/copy.txt
+cezary@cezary-VirtualBox:~/Documents/cpp_workspace$ mkdir Backup
+cezary@cezary-VirtualBox:~/Documents/cpp_workspace$ mv dummy_dir/copy.txt Backup/
+cezary@cezary-VirtualBox:~/Documents/cpp_workspace$ wc -l Backup/copy.txt
```
Comment (BarTes8) [inline homework/linux.txt:7]: wc -l processes.txt not copy.txt

File: homework/linux.txt [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+cezary@cezary-VirtualBox:~/Documents/cpp_workspace$ ps aux >> dummy_dir/process.txt 
+cezary@cezary-VirtualBox:~/Documents/cpp_workspace$ cp dummy_dir/process.txt dummy_dir/copy.txt
+cezary@cezary-VirtualBox:~/Documents/cpp_workspace$ mkdir Backup
+cezary@cezary-VirtualBox:~/Documents/cpp_workspace$ mv dummy_dir/copy.txt Backup/
+cezary@cezary-VirtualBox:~/Documents/cpp_workspace$ wc -l Backup/copy.txt 
+193 Backup/copy.txt
+cezary@cezary-VirtualBox:~/Documents/cpp_workspace$ wc -l Backup/copy.txt >> Backup/selected.txt
+cezary@cezary-VirtualBox:~/Documents/cpp_workspace$ grep /bin dummy_dir/process.txt --line-number > Backup/selected.txt 
+cezary@cezary-VirtualBox:~/Documents/cpp_workspace$ history | tail -n 3 > Backup/commands.txt
+cezary@cezary-VirtualBox:~/Documents/cpp_workspace$ cat Backup/commands.txt Backup/selected.txt
```
Comment (BarTes8) [inline homework/linux.txt:12]: cat selected.txt commands.txt change order

**Review by BarTes8 — COMMENTED** [review]
Change process.txt to processes.txt and then try one more time.

**Comment by ziobron** [comment]
We are closing this PR. If you want to fix your code and get points, please re-open it or raise a new PR

**Pull request closed without merging**